Product vision

louisgosschalk opened the pull request:
Dit is bijna de definitieve visie, opmerkingen worden gewaardeerd. 
crucial needs wordt nog uitgebreid, unique selling points ook. 
de andere punten zijn redelijk in zijn eindfase, maar er komen hier en daar misschien nog wat citaten voor onderbouwing bij.

louisgosschalk committed:
add feedback & latex file

Adds feedback for literature referencing in product vision and latex
generated file for compiling

louisgosschalk committed:
main.tex changes (incl to-do list)

Adds multiple files, generated by latex to compile the correct pdf.
Most important change is in main.tex, in which I added comments of
to-do’s and extended the paragraph about the target user a bit.

louisgosschalk committed:
small changes

Work in progress on description of ADMIRE in target customer

louisgosschalk committed:
Aanmaken van bibliography - work in progress

louisgosschalk committed:
Tussenstap vision

APA style citaties kloppen nu. Het is niet nuttig om alle bestanden te
bekijken, de enige die interessant zijn, zijn de main.tex en eventueel
de pdf.
De andere bestanden zijn automatisch gegenereerd voor latex (behalve de
ref.bib, die heeft alle referenties)

louisgosschalk committed:
uitbreiding customer needs

JensLangerak commented:
Ik zou ons groepsnummer ook op de voorkant neerzetten en volgens mij staat de abstract altijd voor e TOC en hoeft de abstract ook niet in de TOC te staan

JensLangerak commented on docs/product vision/main.tex:
```diff
@@ -35,10 +45,28 @@
+\subsection{persona}
+%persona of wenxin wang
+Person: Wenxin Wang \\
+Wenxin is a trained analyst, has high qualifications \cite{persona}, and is capable of working with moderately complex computer systems. Wenxin likes software to be easy to use and wants results to be clearly visualized, though she is able to work with software which might be a bit more complex and thus a little less easy to use. Wenxin wants minimal repetition of actions so that she does not hav ...
```
Is misschien netter om niet de voornaam te gebruiken

JensLangerak commented on docs/product vision/main.tex:
```diff
@@ -65,28 +106,21 @@ \section{Unique Selling Points}
+%search relatable projects combining multiple data sources (to analyze users)
 \section{Time-frame}
-There's a time-frame of 10 weeks for the entire project. We aim to finish the product in week 9 so we will have the time to debug in week 10. Apart from the debugging, week 10 is very chaotic due to presentations and meetings so there will probably not be a lot of time in that week to develop a lot. We also aim to have a working version continuously throughout the process of development. There is ...
+There's a time-frame of 10 weeks for the entire project. We aim to finish the product in week 9 so we will have the time to debug in week 10. Apart from the debugging, week 10 is very chaotic due to presentations and meetings so there will probably not be a lot of time in that week to develop much more. We also aim to have a working version continuously throughout the process of development. Ther ...
```
Volgens de planning is in week 9 de final input for SIG, dan moet dus de code af zijn

JensLangerak commented:
ziet er goed uit. Denk ook aan de inleidingen. Als ik het me goed herinner hoor je ieder hoofdstuk te beginnen met een korte inleiding waarin staat waar het hoofdstuk over gaat (kan iemand dit bevestigen of tegen spreken, ben hier niet helemaal zeker over), . Eventueel kun je ook het document met een algemene inleiding beginnen, maar dat staat niet specifiek genoemd bij de product vision (wel bij de andere documenten)

louisgosschalk committed:
Volgorde auteurs alfabetisch

louisgosschalk committed:
Revert "Volgorde auteurs alfabetisch"

This reverts commit a22ba75e8d43caed4a6793e0f8d962faf4abb298.

louisgosschalk committed:
Alle github feedback verwerkt en auteurs in alfabetische volgorde

Wenxin = Miss Wang
auteurs alfabetische volgorde
Abstract vóór de TOC
Abstract niet in de TOC
time-frame aangepast

louisgosschalk committed:
Definitieve versie

Deze versie bevat voldoende relevante citaten, de juiste manier van
citeren, APA stijl referenties etc.

Als het goed is, moet dit redelijk final zijn.

Boudewijn26 commented:
Ultieme kudo's voor dit lijstje:
• keep the graphical user interface simple (understandability, operability) 
• make the analysis language small but eﬀective (learnability) 
• create a nice-looking graphical user interface (attractiveness)

Ergens zag ik de zin "The use of the language will is that the analyst wants to make great manipulations with only a couple of commands." Hier lijkt mij dat will weg moet.

In 4.2 stond therefor -> therefore..

Verder ziet het er goed uit.

clanghout commented:
Netjes! :+1:

louisgosschalk committed:
Verwerking commentaar

Boudewijn26 commented:
Ik vind hem goed.

clanghout commented:
Top! ik merge.
